### ISSUE TITLE
Option to enable showing errors to the end user

### ISSUE THREAD

**Issue by seanmcne** [microsoft/sample-app-aoai-chatGPT#319]
We have deployed and been testing the sample, at times we hit capacity and you look in the browser debug tools I can see error messages like this: 
![image](https://github.com/microsoft/sample-app-aoai-chatGPT/assets/2292260/3b5614d2-6a8f-4b45-9f7d-b149d28c5b39)

or a variety of other errors for semantic search capacity, etc.  We expect to see some of these especially as we're testing but it's confusing to some folks when testing as they just see a completely blank response.

Is it possible to enable error messages so our tester knows there was an issue and they can report the issue and error message they're seeing? 

Thanks!

**Comment by sarah-widder** [comment]
Hi @seanmcne these messages are supposed to show in the UI in place of the bot message. What shows in the UI for you when these errors are hit?

**Comment by seanmcne** [comment]
It is completely blank - it shows the 'generating' box and then it disappears. When I looked in the console, I saw the above error message.  
![image](https://github.com/microsoft/sample-app-aoai-chatGPT/assets/2292260/2e75396a-5856-4ef0-b8d8-399d777238f7)

**Comment by sarah-widder** [comment]
Thanks for raising the issue @seanmcne, we will look into fixing the error message display logic. In the meantime, you could try using the gpt-35-turbo-16k model which should let you chat without hitting this error.

**Comment by seanmcne** [comment]
Yes, we already switched models - it was just a strange end user experience that I thought would be good to get resolved. Thanks!
